src/game/ZoneManager.cxx

```diff
@@ -4,6 +4,9 @@
 #include "../services/GameClock.hxx"
 #include "../services/Randomizer.hxx"
 
+#include <algorithm>
+#include <vector>
```

Comment:
Should go to the .hpp file, as there is already the usage of the vector class.

Reply:
you're right